Source/JavaScriptCore/runtime/BufferMemoryHandle.cpp

```diff
@@ -51,13 +51,18 @@ namespace JSC {
 #if ENABLE(WEBASSEMBLY_SIGNALING_MEMORY)
 size_t BufferMemoryHandle::fastMappedRedzoneBytes()
 {
-    return static_cast<size_t>(PageCount::pageSize) * Options::webAssemblyFastMemoryRedzonePages();
+    if (canUseWebAssemblyFastMemory())
+        return static_cast<size_t>(PageCount::pageSize) * Options::webAssemblyFastMemoryRedzonePages();
+    return 0;
```

Comment:
This is not correct. If this code is used, since this is *fast* memory code, it should return the correct size.